schemas/postprocessing/idlparsed.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "https://github.com/w3c/reffy/blob/main/schemas/postprocessing/idlparsed.json",
+
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["spec", "idlparsed"],
+  "properties": {
+    "spec": { "$ref": "../common.json#/$defs/specInExtract" },
+
+    "idlparsed": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["jsNames", "idlNames", "idlExtendedNames", "globals",
+        "exposed", "dependencies", "externalDependencies", "hasObsoleteIdl"],
+      "properties": {
+        "jsNames": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["constructors", "functions"],
+          "properties": {
+            "constructors": { "$ref": "../common.json#/$defs/interfacesByGlobal" },
+            "functions": { "$ref": "../common.json#/$defs/interfacesByGlobal" }
+          }
+        },
+        "idlNames": {
+          "type": "object",
+          "propertyNames": { "$ref": "../common.json#/$defs/interface" },
+          "additionalProperties": {
+            "type": "object",
+            "additionalProperties": true,
+            "required": ["fragment", "type"],
+            "properties": {
+              "fragment": { "type": "string" },
+              "type": { "type": "string" }
```

Comment:
I believe this is same type as the enum defined in idlnames-parsed - maybe worth lifting it to commons?